### ISSUE TITLE
specifying a different base URL

### ISSUE THREAD

**Issue by ahazem** [heartsentwined/ember-auth#2]
Hi,

I was wondering how to go about specifying a different base URL for ajax requests. The thing is, I have two separate apps, the backend is running at `http://localhost:3000` and the frontend is running at `http://localhost:9292`.

At first I tried extending `Auth.RESTAdapter` like one would normally do with any subclass of `DS.RESTAdapter` like so:

``` javascript
App.Adapter = Auth.RESTAdapter.extend({
  url: 'http://localhost:3000'
});
```

but that didn't work as I expected, as it continued sending requests to `http://localhost:9292/users/sign_in` instead of `http://localhost:3000/users/sign_in`.

I later got it working by adding a url property to `Auth.Config` manually and then using it throughout `Auth.signIn` and `Auth.signOut` functions, but I wonder what's the best approach to tackle such case.

**Comment by heartsentwined** [comment]
@ahazem thanks for reporting! This is a bug behavior. I will push a fix in the next release.

**Comment by heartsentwined** [comment]
@ahazem this feature has been implemented in v1.1.0.

``` coffeescript
Auth.Config.reopen
  baseUrl: 'http://localhost:3000'
```

Note that you need to set `Auth.Config.baseUrl` explicitly; `ember-auth` does not inherit the `url` setting from `Auth.RESTAdapter`.

However, I doubt if it will be useful in production. Cross-domain ajax requests is prohobited by default in jQuery. ([ref](http://stackoverflow.com/questions/1933065/ajax-request-to-different-host)) You will need to do further settings on both your servers to enable a direct cross-site ajax.

A better pattern, as suggested in the above SO post, is going through your own server (`localhost:9292` in your case), have your server communicate with the third-party server (`localhost:3000` in your case), and return this third-party content.

**Comment by scottfister** [comment]
This isn't working for me in v9.0.7.

As described in the [docs](http://ember-auth.herokuapp.com/docs) I have set it like so:

```
App.Auth = Ember.Auth.extend({
  baseUrl: 'http://0.0.0.0:3000/v1',
  ...
});
```

I also tried (with and without `Config`)

```
App.Auth.Config.reopen({
  baseUrl: 'http://0.0.0.0:3000/v1'
});
```

But each time it still defaults to `http://localhost:9000/` when I try to perform the `this.auth.signIn` method.
